docs/08_dl_dnn.md

```diff
@@ -1241,10 +1239,10 @@ cv_fitted %>%
 #> # A tibble: 4 × 4
 #>   .metric      mean     n  std_err
 #>   <chr>       <dbl> <int>    <dbl>
-#> 1 accuracy    0.819     5 0.000523
```

Comment:
Is it expected that we are getting different results here?

Reply:
It's the same as Ch 9 and 10, right? That the seeds aren't really reproducible into the Python process?

Reply:
I made a mistake. Please refer to the other comment